src/sdk.ts

```diff
@@ -46,12 +46,10 @@ export function init<O>(
       iteratee: (frame: StackFrame) => {
         if (frame.filename) {
           const isReachableHost = /^https?:\/\//.test(frame.filename);
-
           frame.filename = frame.filename
-            .replace(/^https?:\/\/localhost(:\d+)?/, '')
-            .replace(/^ng:\/\//, '')
-            .replace(/^capacitor:\/\/localhost(:\d+)?/, '');
-
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any, no-restricted-globals
+            .replace((window as any).WEBVIEW_SERVER_URL, '')
+            .replace(/^ng:\/\//, '');
```

Comment:
would the following work for you?

```typescript
interface GlobalObject  extends Window{
  WEBVIEW_SERVER_URL?: string;
};

/**
 * Return the current webview url if found.
 */
export function getCurrentServerUrl(): string | undefined {
    return (self as GlobalObject).WEBVIEW_SERVER_URL;
}

```
this code could be added on src/tools/webViewUrl.ts

and just as a safe rule, we could use the old rule
```typescript
            .replace(/^https?:\/\/localhost(:\d+)?/, '')
            .replace(/^capacitor:\/\/localhost(:\d+)?/, '');
            .replace(/^ng:\/\//, '')
```
if the return of getCurrentServerUrl was undefined, else we could use the new method with `.replace(/^ng:\/\//, '')`.

Reply:
That could work too. But my sentry trial expired, so I can’t test the proposed changes.

Reply:
I have tested it and it is working, if you prefer I can merge this PR to another branch and do the final changes before going to the main branch

Reply:
despite the trial ended, I could still test with the free plan, I've updated with the requested changes